src/lib.rs

```diff
@@ -549,6 +561,7 @@ enum Response<'a> {
     Ok,
     Error(u8),
     String(&'a str),
+    StringAsString(String),
```

Comment:
I'd just mash these two up into `String(Cow<'a, str>)`.

Reply:
Yeah, that's a good idea.  I will fix it.

Reply:
I can do it this way, but be sure to look at the multi-threading patches.  Those remove all the lifetimes here so that `Command` can be `Send`.